# Request / response service to remove all users from from the device.

---